webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -659,7 +661,7 @@ const AdminDefinition: AdminDefinitionType = {
                 url: 'system_attributes/attribute_based_access_control',
                 title: defineMessage({id: 'admin.sidebar.attributeBasedAccessControl', defaultMessage: 'Attribute-Based Access'}),
                 isHidden: it.any(
-                    it.not(it.licensedForSku(LicenseSkus.EnterpriseAdvanced)),
+                    it.not(it.any(it.licensedForSku(LicenseSkus.EnterpriseAdvanced), it.licensedForSku(LicenseSkus.Entry))),
```

Comment:
This we should probably replace with ` minLicenseTier` that does the level check instead